test.py

```diff
@@ -41,8 +41,8 @@ def set_chrome_options() -> None:
     Sets chrome options for Selenium.
     """
     chrome_options = Options()
-    # Comment out next line if you want to run tests without opening browser.
-    # It's useful if you run it in container.
+    # Comment out next line if you want to run tests with a opening browser.
+    # GUI is useless when running containers.
     chrome_options.add_argument("--headless")
```

Comment:
if you want to have option to run chrome in headless mode or not, I would add flag --headless